components/cards/FriendsSkeleton.tsx

```diff
@@ -0,0 +1,83 @@
+import { theme } from "@/theme"
+import { useCallback, useEffect } from "react"
+import { View, StyleSheet } from "react-native"
+import Animated, {
+  withRepeat,
+  withSequence,
+  withTiming,
+  useAnimatedStyle,
+  useSharedValue,
+} from "react-native-reanimated"
+
+export const FriendsSkeleton = () => {
+  const opacity = useSharedValue(0.5)
+  const createAnimatedLoader = useCallback(() => {
```

Comment:
```suggestion
  const animateLoader = useCallback(() => {
```
"createAnimatedLoader" sounds like a function that also mounts something on the screen